src/Stratis.Bitcoin.Features.Consensus/ConsensusFeature.cs

```diff
@@ -209,7 +209,7 @@ public static IFullNodeBuilder UseStratisConsensus(this IFullNodeBuilder fullNod
                         services.AddSingleton<IConsensusLoop, ConsensusLoop>();
                         services.AddSingleton<StakeChainStore>().AddSingleton<StakeChain, StakeChainStore>(provider => provider.GetService<StakeChainStore>());
                         services.AddSingleton<IStakeValidator, StakeValidator>();
-                        services.AddSingleton<ConsensusManager>().AddSingleton<INetworkDifficulty, ConsensusManager>();
+                        services.AddSingleton<ConsensusManager>().AddSingleton<INetworkDifficulty, ConsensusManager>().AddSingleton<IGetUnspentTransaction, ConsensusManager>();
```

Comment:
Not sure I would put all this on the same line.

Reply:
Fixed